tests/Browser/UserTest.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use App\User;
+class UserTest extends DuskTestCase
+{
+    /**
+     * Permisson account admin
+     *
+     * @return $admin
+     */
+    public function accountAdmin()
+    {
+        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+
+        return $admin;
+    }
+
+    /**
+     * Get data from cogfig to test.
+     * 
```

Comment:
Remove blank at the end line.
Search and replace like this "     * "

Reply:
i fixed

Reply:
Not fixed yet